scripts/ci/krb5/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM gerardo/airflow-testing:latest
```

Comment:
We can't have the testing suite depend on a private docker-repo. Try to get this image hosted somewhere official like docker-repos.

Reply:
Yeah, definitely agree on that. The idea is for the Airflow project to also have an `incubator-airflow-ci` repo as well, that generates this image semi-regularly under the `apache/airflow-testing` docker registry namespace, just like CouchDB and a few other Apache projects do https://travis-ci.org/apache/couchdb-ci.

The repo that generates that image lives under https://github.com/gerardo/incubator-airflow-ci, so you could consider it as part of this PR.